src/test/java/gmbs/model/car/vo/NameTest.java

```diff
@@ -0,0 +1,42 @@
+package gmbs.model.car.vo;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class NameTest {
+
+    private static final String OVER_THAN_MAX_LENGTH = "SixLen";
+    private static final String BLANK = " ";
+
+    @DisplayName("정상적인 이름이 들어오면 생성에 성공한다")
```

Comment:
정상적인 이름이 어떤 것인지 명시해주는 것이 더 좋을 것 같습니다.

```suggestion
    @DisplayName("이름의 길이가 5자 이하여야 생성에 성공한다.")
```